src/modules/vtol_att_control/standard.cpp

```diff
@@ -182,7 +182,7 @@ void Standard::update_vtol_state()
 		} else if (_vtol_schedule.flight_mode == vtol_mode::TRANSITION_TO_FW) {
 			// continue the transition to fw mode while monitoring airspeed for a final switch to fw mode
 			if (((_params->airspeed_disabled ||
-			      _airspeed->indicated_airspeed_m_s >= _params->transition_airspeed) &&
```

Comment:
@sfuhrer I think we need to check for airspeed validity here as well. I assume we can just remove the check on "_params->airspeed_disabled" and check if airspeed is finite, correct?

Reply:
Same holds for everywhere else in the vtol code.

Reply:
I've added a PX4_ISFINITE(airpseed) everywhere airspeed is used, and if it's invalid it does the same as when "_params->airspeed_disabled". I've not yet removed the parameter, not 100% if we should do so with this PR, or if that's fuel for another PR (I was also thinking about removing the CBRK_AIRSPD_CHECK, but this then needs some further thoughts.